### DISCUSSION TITLE
Locating LEGUP.jar file

### DISCUSSION THREAD

**Discussion by Elambeth** [Bram-Hub/LEGUP#804] [Q&A]
Hello, I am new here and would love to try out the tool, I have read the wiki/set-up, but I can't seem to locate the LEGUP.jar file.
Please excuse my technical incompetence 
Thanks

**Comment by jadeandtea** [comment]
Hello! Sorry for the late response. If you go to the [releases](https://github.com/Bram-Hub/LEGUP/releases) tab and download the zip file corresponding to your operating system, the LEGUP.jar file should be in that zip.
